### ISSUE TITLE
grey_replaceText() doesn't send UITextFieldTextDidChangeNotification nor UIControlEventEditingChanged

### ISSUE THREAD

**Issue by Arcank** [google/EarlGrey#253]
grey_replaceText() doesn't send UITextFieldTextDidChangeNotification

When using grey_replaceText() on a UITextField, it should send the UITextFieldTextDidChangeNotification notification.

As of today:
https://github.com/google/EarlGrey/blob/38b490e5a156f16a00dd7777b4c4cdcffe6a4e24/EarlGrey/Action/GREYActions.m#L332
But setText: doesn't send the notification.

You might send the notification yourself:
```
      [element setText:text];
      NSNotification *notification  = [NSNotification notificationWithName:UITextFieldTextDidChangeNotification object:element];
      [NSNotificationCenter.defaultCenter postNotification:notification];
```

**Comment by bootstraponline** [comment]
Nice. Want to send a pull request?

**Comment by Arcank** [comment]
Dummy project reproducing:
[TestUITextFieldNotifications.zip](https://github.com/google/EarlGrey/files/462136/TestUITextFieldNotifications.zip)

**Comment by tonlen** [comment]
> Dummy project reproducing:
> [TestUITextFieldNotifications.zip](https://github.com/google/EarlGrey/files/462136/TestUITextFieldNotifications.zip)



> grey_replaceText() doesn't send UITextFieldTextDidChangeNotification
> 
> When using grey_replaceText() on a UITextField, it should send the UITextFieldTextDidChangeNotification notification.
> 
> As of today:
> [EarlGrey/EarlGrey/Action/GREYActions.m](https://github.com/google/EarlGrey/blob/38b490e5a156f16a00dd7777b4c4cdcffe6a4e24/EarlGrey/Action/GREYActions.m#L332)
> 
> Line 332 in [38b490e](/google/EarlGrey/commit/38b490e5a156f16a00dd7777b4c4cdcffe6a4e24)
> 
>  [element setText:text]; 
> 
> But setText: doesn't send the notification.
> You might send the notification yourself:
> 
> ```
>       [element setText:text];
>       NSNotification *notification  = [NSNotification notificationWithName:UITextFieldTextDidChangeNotification object:element];
>       [NSNotificationCenter.defaultCenter postNotification:notification];
> ```

UITextView doesn't send the UITextViewTextDidChangeNotification notification

**Comment by bootstraponline** [comment]
you'll want to open a new issue instead of commenting on an old issue from 2016.

If you have time, sending a PR to add the notification would be great.

**Comment by khandpur** [comment]
+1. Please open new issue and would appreciate if you can send a PR along :)

**Comment by tirodkar** [comment]
@tonlen This is already fixed in EarlGrey 2.0. Please take a look at 

https://github.com/google/EarlGrey/blob/earlgrey2/AppFramework/Action/GREYActions.m
